### PR TITLE
Adding metadata preservation tutorial notebooks

### LINKED ISSUE #16: Add description for metadata preservation tutorial
https://insightsoftwareconsortium.github.io/GetYourBrainStraight/HCK01_2022_Virtual/Tutorials/

### PR DESCRIPTION
**Pull request by dzenanz**
Closes #16.

### CODE REVIEW

**Comment by thewtex** [comment]
Can you add a MyBinder link in the tutorial description Markdown? This will help testing / review.

**Comment by dzenanz** [comment]
Is MyBinder down currently? Or am I doing something wrong? Tail or status outuput:
```text
Step 48/48 : CMD ["jupyter", "notebook", "--ip", "0.0.0.0"]
 ---> [Warning] Your kernel does not support swap limit capabilities or the cgroup is not mounted. Memory limited without swap.
 ---> Running in b31f8c21fb9d
Removing intermediate container b31f8c21fb9d
 ---> 4b94e35c155d
{"aux": {"ID": "sha256:4b94e35c155d8145d934142e20278cfdaa6f523e2289bdc907fc99657f70a782"}}Successfully built 4b94e35c155d
Successfully tagged gesiscss/binder-r2d-g5b5b759-insightsoftwareconsortium-2dgetyourbrainstraight-ff41d1:ea668a498e5bc94f2fe62303cf48483b4b084fb8
Pushing image
Pushing image
Pushing image
Pushing image
Pushing image
Successfully pushed gesiscss/binder-r2d-g5b5b759-insightsoftwareconsortium-2dgetyourbrainstraight-ff41d1:ea668a498e5bc94f2fe62303cf48483b4b084fb8Built image, launching...
Launching server...
Server requested
2022-03-31T18:19:28.240109Z [Normal] Successfully assigned bhub-ns/jupyter-insightsoftware-2durbrainstraight-2dfmm7fs8c to spko-css-app03
2022-03-31T18:19:31Z [Normal] Container image "jupyterhub/k8s-network-tools:1.1.2" already present on machine
2022-03-31T18:19:31Z [Normal] Created container block-cloud-metadata
2022-03-31T18:19:31Z [Normal] Started container block-cloud-metadata
2022-03-31T18:19:32Z [Normal] Pulling image "gesiscss/binder-r2d-g5b5b759-insightsoftwareconsortium-2dgetyourbrainstraight-ff41d1:ea668a498e5bc94f2fe62303cf48483b4b084fb8"
2022-03-31T18:19:33Z [Warning] Failed to pull image "gesiscss/binder-r2d-g5b5b759-insightsoftwareconsortium-2dgetyourbrainstraight-ff41d1:ea668a498e5bc94f2fe62303cf48483b4b084fb8": rpc error: code = Unknown desc = Error response from daemon: pull access denied for gesiscss/binder-r2d-g5b5b759-insightsoftwareconsortium-2dgetyourbrainstraight-ff41d1, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
2022-03-31T18:19:33Z [Warning] Error: ErrImagePull
2022-03-31T18:19:35Z [Normal] Back-off pulling image "gesiscss/binder-r2d-g5b5b759-insightsoftwareconsortium-2dgetyourbrainstraight-ff41d1:ea668a498e5bc94f2fe62303cf48483b4b084fb8"
2022-03-31T18:19:35Z [Warning] Error: ImagePullBackOff
```

**Comment by thewtex** [comment]
Looking great!

**Comment by satra** [comment]
just a thought here. if you install [jupytext](https://jupytext.readthedocs.io/en/latest/install.html) you can mark the notebook to save as both ipynb and markdown, which may make diffs easier to check.

File: HCK01_2022_Virtual/Tutorials/MetadataPreservation/ITK_image_grid.ipynb [hunk 1]
```diff
@@ -0,0 +1,530 @@
...
+    "# Load image, show its basic metadata and plot a slice\n",
+    "image = itk.imread(files_and_hashes[0][0])\n",
+    "image2D = itk.imread(files_and_hashes[1][0])\n",
+    "\n",
+    "print(f\"Dimension: {image.GetImageDimension()}\")\n",
+    "print(f\"Spacing: {image.GetSpacing()}\")\n",
+    "print(f\"Origin: {image.GetOrigin()}\")\n",
+    "size = image.GetLargestPossibleRegion().GetSize()\n",
+    "print(f\"Size: {size}\")\n",
+    "print(f\"LPR index: {image.GetLargestPossibleRegion().GetIndex()}\")\n",
```
[the remark this answers is not in the document]
Comment (dzenanz) [reply HCK01_2022_Virtual/Tutorials/MetadataPreservation/ITK_image_grid.ipynb:160]: LPR = LargestPossibleRegion, but I will change it to avoid ambiguity.

**Review by thewtex — APPROVED** [review]

**Pull request merged**